### ISSUE TITLE
[REQUEST] puppeteer OB2 ? 

### ISSUE THREAD

**Issue by beer15** [openbullet/openbullet#721]
Hey, 

is it possible or is it too complicated to add puppeteer to openbullet? i know it's a lot of work.

thanks

**Comment by FlySolutions** [comment]
good idea, puppeteer is way better than selenium. only thing is that it is is NodeJS and i don't think that the c# port of it is that good.

**Comment by lapidoria** [comment]
Puppeteer in c#
https://github.com/hardkoded/puppeteer-sharp

**Comment by N3ars** [comment]
Or adding CapserJS+PhantomJS could be great too ! 
Tbh we need something that can emulate and control webpage properly

**Comment by openbullet** [comment]
OB2 uses puppeteer instead of selenium.
